apps/nextjs-website/src/config.ts

```diff
@@ -8,6 +8,9 @@ See BrowserConfig.ts and BrowserEnv.ts as examples.
 // TODO: Add environment parser
 export const docsPath = process.env.PATH_TO_GITBOOK_DOCS;
 export const cookieDomainScript = process.env.NEXT_PUBLIC_COOKIE_DOMAIN_SCRIPT;
+export const matomoScriptSrc =
+  process.env.NEXT_PUBLIC_MATOMO_SCRIPT_SRC ||
+  'https://cdn.matomo.cloud/pagopa.matomo.cloud/container_cT0RWuHZ.js';
```

Comment:
Is this fallback still necessary?

```suggestion
  process.env.NEXT_PUBLIC_MATOMO_SCRIPT_SRC;
```

Reply:
we don't know if the variable will be available once deployed (like last time on DEV). i suggest to keep this here for now to avoid loosing all the tracking on prod